include/seastar/http/request.hh

```diff
@@ -148,6 +148,16 @@ struct request {
             return it == _headers.end() || !case_insensitive_cmp()(it->second, "close");
         }
     }
+
+    /**
+     * Set the query parameters in the request objects.
+     * query param appear after the question mark and are separated
+     * by the ampersand sign
+     */
+    sstring parse_query_param();
```

Comment:
What's returned? It can't be guessed from the function name.

Reply:
It's an ex-set_query_param() method of the https, it returns path part of the URL. Will add